### PR TITLE
Login and SignUp Activity added.

### LINKED ISSUE #8: Login Activity
Creating a Login Activity via email and password connected through firebase.

### PR DESCRIPTION
**Pull request by Sarthak2601**
Fixes #8 and #9 

Changes: 
---- Code Additions -----
-Integrated Firebase in the App.
-Sign Up through Email and Password.
-Log-in Activity through Email and Password.
-Log-out Button Added in HomeActivity for Signing out. 
-Also added the feature which checks if the user has logged in the app in past and logs him/her in straightaway. 
---- UI Connections ----- 
*Connected Splash Screen with Log-in Activity.
*Connected Log-in Activity with Sign-Up Activity.
*Connected Log-in Activity with Home Activity. 
*Connected Home Activity with Log-in Activity when Log-out button is clicked.

Screenshots for the change:
Screenshot_2019-12-03-16-52-13-812_io github njackwinterofcode robotalk
Screenshot_2019-12-03-16-52-19-173_io github njackwinterofcode robotalk
Screenshot_2019-12-03-16-52-38-555_io github njackwinterofcode robotalk

### CODE REVIEW

**Comment by somenath1435** [comment]
@Sarthak2601 Add somenath1435@gmail.com in firebase

File: app/src/main/java/io/github/njackwinterofcode/robotalk/LoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class LoginActivity extends AppCompatActivity {
+
+
```
Comment (somenath1435) [inline app/src/main/java/io/github/njackwinterofcode/robotalk/LoginActivity.java:21]: Remove unnecessary spaces

File: app/src/main/java/io/github/njackwinterofcode/robotalk/LoginActivity.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        textView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                startActivity(new Intent(LoginActivity.this, signUp.class));
+            }
+        });
+        button.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+
```
Comment (somenath1435) [inline app/src/main/java/io/github/njackwinterofcode/robotalk/LoginActivity.java]: there are many unncessary spaces in your PR, remove all

File: app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        app:layout_constraintVertical_bias="0.306"
+        android:drawableStart="@drawable/ic_lock_black_24dp"
+        android:drawablePadding="5dp"
+        />
+
+    <Button
+        android:id="@+id/button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Log-in"
```
Comment (somenath1435) [inline app/src/main/res/layout/activity_login.xml]: use strings.xml instead of hardcoding

**Review by somenath1435 — CHANGES_REQUESTED** [review]

**Comment by somenath1435** [comment]
@Sarthak2601 Still some files contain unnecessary spaces/lines. Remove all

**Comment by Sarthak2601** [comment]
> @Sarthak2601 Add [somenath1435@gmail.com](mailto:somenath1435@gmail.com) in firebase

Do you mean adding this email account to the Auth database so that you can log in? If yes then you can make it through Sign up Activity. If otherwise, please elaborate.

**Comment by Sarthak2601** [comment]
@somenath1435 Changes done.

**Pull request merged**